Topics/Development_Process/Azure_webapp_deployment_with_nodejs.md

```diff
@@ -1,10 +1,191 @@
+
+Azure is a [cloud service provider](#azure-terminology) by Microsoft that provides extensive services, including application hosting, messaging, data, storage, and more, for developers to leverage when building applications. 
+
+It is one of the most popular [cloud service providers](#azure-terminology), and if you're an 18+ year old student, you can get $100 in Azure credit with the [Github Student Developer Pack](https://education.github.com/pack). 😁
+
+## Getting Started
+
+Before being able to work with Azure, you must first create an account [here](https://azure.microsoft.com/en-ca/get-started/azure-portal). As mentioned earlier, after creating your account, you can get $100 in Azure credit with the [Github Student Developer Pack](https://education.github.com/pack) if you're an 18+ year old student.
+
+After creating an account, make sure you have an Azure subscription (such as Azure for Students above) to use.
+
+## Installation
+
+The Azure Portal is simple, and easy to use to interact with and create Azure services. Aside from your Azure account, no other setup is necessary.
+
+For the examples in this article, you'll also require the Azure CLI. Instructions to install it on your computer can be found [here](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) 
+
+Although not covered in this article, other alternatives for interacting with Azure can be found [here](https://learn.microsoft.com/en-us/azure/developer/intro/azure-developer-create-resources).
+
+While we won't be going into too much detail here, you will need to have Node.js and npm installed to follow along and get a basic Node.js web app with Express running (install Node.js and npm together for your system [here](https://nodejs.org/en/download/current)). If you're working with another tech stack, links to help you find appropriate documentation and examples will be provided.
+
+## Deployment
+
+Azure has a variety of ways to host your application, including Azure App Service, Azure Functions, Azure Container Instances, Azure Kubernetes Service, and more, which you can choose to suit your specific usecase. 
+
+See more about the different kinds of hosting options and reasons to choose one over the other [here](https://learn.microsoft.com/en-us/azure/developer/intro/hosting-apps-on-azure).
+
+The following will detail how one would deploy a Node.js Web App using Azure App service, and integrate it with Github Actions for CI/CD.
+
+### Azure App Service
+
 Azure application service allows you to build and host web applications, mobile backends and RESTful APL in the selected programming language without managing the infrastructure. It provides automatic scaling and high availability, supports Windows and Linux, and supports automatic deployment from GitHub, Azure Devops or any Git repository. You can refer to our quick start, tutorials and examples to learn how to use Azure application services.
```

Comment:
API, not APL.

You can also perhaps rework the "You can refer to our quick start, tutorials and examples to learn how to use Azure application services." to better match the what you wrote around this paragraph.

Reply:
Sounds good, thanks, I missed that since I was trying to reuse the content that was already in this file. Thanks, this will make the article as a whole feel more cohesive!